static/demo/layer-demo.html

```diff
@@ -80,8 +80,9 @@
 </head>
 <body>
     <div style="text-align: center;">
-        <h2>MapML Demo: &lt;layer-&gt;</h2>
-        <button class="run" onclick="plot()">▶</button>
+      <!------------------------------------------------- CHANGE THIS ------------------------------------------------->
+      <h2>MapML Demo: &lt;layer-&gt;</h2>
+      <button class="run" onclick="plot()">▶</button>
```

Comment:
I haven't tested this using a screen reader: `<button class="run" onclick="plot()">▶</button>`, is it announced correctly?

https://scottvinkle.me/blogs/work/svg-and-emoji-icon-accessibility#accessible-emoji:

> Take for instance the common phrases, "I ❤️ you" – announced as, "I red heart you."

Perhaps these buttons should be changed to:

```diff
<button class="run" onclick="plot()"
+ aria-label="run">
- ▶
+ <span aria-hidden="true">▶</span>
</button>
```